### PR TITLE
gnrc_uhcp, gnrc_dhcpv6_client_6lbr: disable Router Advertisements on upstream interface

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description

We don't want to advertise ourselves as a router to the upstream router.
This also has the side effect the the node / border router will not request a prefix from the upstream router, so it can take very long to obtain a valid prefix on an interface with `RTR_AVD` enabled.

### Testing procedure

Run `examples/gnrc_border_router` on a network were a router is advertised on the upstream interface (either by running `radvd` manually or by connecting to a WiFi network where routes are advertised).

You should immediately get a global address on the upstream interface.

Previously this would take several minutes.

### Issues/PRs references
#13668 if you want to use `radvd` locally

### CODE REVIEW

**Comment by miri64** [comment]
> This also has the side effect the the node / border router will not request a prefix from the upstream router, so it can take very long to obtain a valid prefix on an interface with `RTR_AVD` enabled.

Huh? Prefixes are requested via router solicitation. Why would deactivating the router advertisements make this longer?

**Comment by benpicco** [comment]
Sorry my wording was bad.
This side effect happened *before* this patch.
This PR fixes it.

**Comment by miri64** [comment]
> Sorry my wording was bad.
> This side effect happened _before_ this patch.
> This PR fixes it.

Ah, yeah that makes sense ^^"

File: sys/net/gnrc/application_layer/dhcpv6/client_6lbr.c [hunk 1]
```diff
@@ -85,6 +86,9 @@ static void _configure_upstream_netif(gnrc_netif_t *upstream_netif)
         addr.u8[15] = 2;
         gnrc_netif_ipv6_addr_add(upstream_netif, &addr, 64, 0);
     }
+
+    /* disable router advertisements on upstream interface */
```
Comment (miri64) [inline sys/net/gnrc/application_layer/dhcpv6/client_6lbr.c]: Here and below: Some more in-depth reasoning would be nice. How about something like this?

```suggestion
    /* disable router advertisements on upstream interface. With this, the border
     * router
     * 1. Does not confuse the upstream router to add the border router to its 
     *    default router list and
     * 2. Solicits upstream Router Advertisements quicker to auto-configure its
     *    upstream global address.
     */
```

**Comment by miri64** [comment]
Mhhhhhhhhhh.... With `ENABLE_DEBUG=1` in `sys/net/gnrc/network_layer/ipv6/nib/nib.c` I get:

```
nib: Received valid router advertisement:
     - Source address: fe80::682e:4fff:fe3d:dfca
     - Destination address: fe80::2
     - Cur Hop Limit: 64
     - Flags: --
     - Router Lifetime: 1800s
     - Reachable Time: 0ms
     - Retrans Timer: 0ms
nib: default router list is full. Ignoring RA from fe80::682e:4fff:fe3d:dfca
```

and no prefix :-(

**Comment by miri64** [comment]
Ah, your use case has `CONFIG_GNRC_DHCPV6_CLIENT_6LBR_STATIC_ROUTE` unset. Let me just increase the size.

**Comment by miri64** [comment]
Please squash!

**Pull request merged**